### DISCUSSION TITLE
Sharing community related files across organization repositories

### DISCUSSION THREAD

**Discussion by mabar** [community/community#9110] [Other Feature Feedback, Questions, & Ideas]
Github asks me to finish [community profile](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/about-community-profiles-for-public-repositories) and add some files like Code of conduct, Contributing, Issue templates, etc. to each repository.

I want to add them but I don't like the idea of copy-pasting these files and updates to them into ever-growing number of repositories as I want them to be exactly same for every single of my repositories. 

I would like to ask whether we could have a centralized location for these files in the special [.github](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) repository to lower maintenance cost and make the process less error-prone. 

Thank you

**Comment by nul800sebastiaan** [comment]
This is currently possible: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

However.. 🙈 I found that the code of conduct link on my newly created repository works great, until I make a different branch my default and then the link also changes to that branch name.

So in my `.github` repo I only have a `main` branch. I created my new repository, pushed some code and then changed the default branch to `dev` as that's where I want contributions to go by default. Now the Code of Conduct link expects a `dev` branch in the `.github` repo as well. 

That's all well and good, but for another repository I have a `v9/contrib` branch as the default, another team is still on `master`.. It's not feasible for me to create all these branches on the central `.github` repository and push the exact same code of conduct to all of the branches, not knowing what my colleagues will name their default branches in the future. 

So.. nice feature, but it doesn't make sense for us unfortunately.

**Reply by martinwoodward** [reply]
@nul800sebastiaan - the .github repo should pick up the default branch settings from that repo and your org wide default:

![image](https://user-images.githubusercontent.com/856858/158992134-755fcb4e-d62d-4178-bf4f-1ea3f2ccba0e.png)

Can you confirm that your org wide default is set to `main` along with the repo default for the `.github` repo being set to `main`?

**Reply by nul800sebastiaan** [reply]
Ha! That's weird, it's working now! 
I guess it was my first push that took a few minutes to propagate, so now it works exactly as expected, no problem any more. 

Thanks for the quick reply, much appreciated! 🙏

**Comment by benoittgt** [comment]
I would be great with `AGENTS.md` file.

**Comment by HUNTERcrackX** [comment]
i found this on dark web


http://pojxcfh2b7dqb3b7qwijfrzwzpm5psdxhuc7vni2wf4kmsw2jvru2pid.onion
